### PR TITLE
add description of new bulk collection options

### PR DESCRIPTION
**Pull request by pretendWhale**
Adds description of new bulk collection options created in https://github.com/MarkUsProject/Markus/pull/6341

### CODE REVIEW

**Comment by pretendWhale** [comment]
Thanks - those are great changes. One minor thing I was a little bit ambivalent about:

I added a note that the 'collect recent submissions' box can be used to collect before the due date. I think this probably makes sense given the 'default' behavior prevents early collection, and the modal text doesn't make it obvious, but I'm not sure I love the wording I used (and maybe I'm wrong and it isn't needed!). What do you think?

**Review by mishaschwartz — APPROVED** [review]

**Comment by mishaschwartz** [comment]
@pretendWhale I think that the wording is fine for the "collect recent submissions" checkbox. I think that it's important to make it clear that submissions can now be collected before the due date. 

The only thing that I might change is instead of referring to due date/grace period, we could refer to the "due date plus any additional late submission penalty period" (or maybe something a little less wordy if you can). The only reason is that we only really use the term "grace period" to refer to the policy that removes grace credits, not the other late submission policies.

In fact, it might just be fine to change it to "due date" without any reference to the extra time since that seems to be what we're doing elsewhere in the wiki.

**Review by david-yz-liu — COMMENTED** [review]
@pretendWhale sorry I forgot to mention this, but please update the description and screenshot with the updated UI options

File: Instructor-Guide--Assignments--Marking--Set-Up.md [hunk 1]
```diff
@@ -46,16 +46,17 @@ Before you may being grading an assignment, you must first collect the submitted
...
+The default option when the collect command is executed will select the file version for each submission that meets the due date and/or late penalty set for this assignment. It will not recollect submissions that have previously been collected. It then creates the views for annotating and grading each submission.
 
 However, the modal offers a few additional options for collection:
 
-1. **Recollect previously collected submissions** - if this box is checked, MarkUs will recollect submissions that have previously been collected. **WARNING**: If submissions with grading work are recollected, that work will be overwritten.
-2. **Collect most recent submissions** - if this box is checked, MarkUs will collect the most recent version of files the student has submitted, regardless of the due date and grace periods. This can be used to collect submissions both before and after the due date/grace credit periods.
-3. **Apply Late Penalty** - this box is checked by default. If it is unchecked, MarkUs will not apply penalties or deduct grace credits for submissions created after the due date.
```
Comment (mishaschwartz) [inline Instructor-Guide--Assignments--Marking--Set-Up.md]: This is still an option that needs to be described. It only shows up if collecting most recent, so please explain that as well in your description. 

Also, please include a screenshot of what the modal looks like with this options visible

**Review by mishaschwartz — CHANGES_REQUESTED** [review]

**Review by mishaschwartz — APPROVED** [review]

**Pull request merged**